samples/drivers/memc/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _memc:
+
+MEMC Driver Sample
+##################
+
+Overview
+********
+
+This sample can be used with any memory controller driver that
+memory maps external RAM. It is intended to demonstrate
+the ability to read and write from the memory mapped region.
+
+Note that the chosen region (set by zephyr,sram-ext) chosen node should not
```

Comment:
This is an `alias` instead of a `chosen` node.

Reply:
It is, sorry- changed that right before pushing and forgot to update the documentation. Will fix that now